package.json

```diff
@@ -8,7 +8,9 @@
     "@farcaster/fishery": "^2.2.3",
     "@farcaster/hub-nodejs": "^0.10.21",
     "ethers": "^6.10.0",
+    "next": "13.5.6",
```

Comment:
Is it `next` necessary for this on? I will guess `react-dom` is the only really dependency need it.